Ports/zlib/package.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=zlib
-version=1.2.13
+version=1.3
 useconfigure=true
```

Comment:
While we are here, let's update this to match the newer styling guidelines:

```suggestion
port='zlib'
version='1.3'
useconfigure='true'
```